### PR TITLE
Review slandelle

### PR DESCRIPTION
**Pull request by slandelle**
toc-users

### CODE REVIEW

**Comment by notdryft** [comment]
If /developer no longer exists, you need to make sure the redirectTo config point to something that exists (`config/_default/params.toml`)

File: content/docs/user/artifact_gen/index.md [hunk 1]
```diff
@@ -12,20 +12,27 @@ menu:
...
-When you'll be deploying tests with FrontLine, it will replace your Gatling OSS dependencies with their custom counterparts.
+FrontLine deploys packages containing your compiled Simulations and resources.
+Those packages have to be generated upstream.
+FrontLine is compatible with Gatling 3.3, 3.4 and 3.5.
 
-## Configuring Gatling Projects
+### Gatling zip bundle
 
-### Maven
+Please copy the `artifact.sh` or `artifact.bat` files in the `bin` directory of your Gatling unzipped bundle.
```
Comment (Isammoc) [inline content/docs/user/artifact_gen/index.md:23]: Direct links?

```suggestion
Please copy the [`artifact.sh`](https://raw.githubusercontent.com/gatling/gatling/master/gatling-bundle/src/universal/bin/artifact.sh) or [`artifact.bat`](https://raw.githubusercontent.com/gatling/gatling/master/gatling-bundle/src/universal/bin/artifact.bat) files in the `bin` directory of your Gatling unzipped bundle.
```

File: content/docs/user/artifact_gen/index.md [hunk 2]
```diff
@@ -90,9 +97,11 @@ In your `pom.xml`, you have to add in:
 </build>
 ```
 
-You can run `mvn package -DskipTests` in your terminal and check you get a jar containing all the dependencies of the simulation.
+Please run the `mvn clean package -DskipTests` command  in your terminal and generate the `target/<artifactId>-<version>-shaded.jar` file.
+You'll have to upload this file in the new [Artifacts section](/docs/user/artifacts).
 
-You can also exclude dependencies you don't want to ship, eg:
+{{< alert ip >}}
```
Comment (notdryft) [inline content/docs/user/artifact_gen/index.md:103]: tip?

File: content/docs/user/artifact_gen/index.md [hunk 3]
```diff
@@ -12,20 +12,27 @@ menu:
...
-## Configuring Gatling Projects
+### Gatling zip bundle
 
-### Maven
+Please copy the `artifact.sh` or `artifact.bat` files in the `bin` directory of your Gatling unzipped bundle.
+Those files haven't been bundled into an official Gatling release yet, but you can download them from [the Gatling repository on GitHub](https://github.com/gatling/gatling/tree/master/gatling-bundle/src/universal/bin).
 
-In your `pom.xml`, you have to add in:
+Please run the script matching your operating system and generate the `target/artifact.jar` file.
+You'll have to upload this file in the new [Artifacts section](/docs/user/artifacts).
```
Comment (Isammoc) [inline content/docs/user/artifact_gen/index.md:27]:
```suggestion
You'll have to upload this file in the new [Artifacts section](/docs/user/artifacts_conf).
```

**Review by notdryft — CHANGES_REQUESTED** [review]

**Comment by slandelle** [comment]
PTAL again

**Review by Isammoc — APPROVED** [review]

**Review by notdryft — APPROVED** [review]

**Pull request merged**